build-container/entrypoint.sh

```diff
@@ -68,6 +77,8 @@ failover() {
             set -x
             tmp_file="$(mktemp)"
             curl -f -o "${tmp_file}" -L "${FAILOVER_BASE_URL}/${pkgfile}"
+            curl -f -o "${tmp_file}.sig" -L "${FAILOVER_BASE_URL}/${pkgfile}.sig"
+            gpg --verify "${tmp_file}.sig" "${tmp_file}"
             sudo mv "${tmp_file}" "/scratch/.buildroot/root/repo/${pkgfile}"
```

Comment:
I wonder if we don't want to first check that all files in the failover verify correctly and only then copy everything to the repo. This would prevent possible partially fetched failovers. On the other hand, I'm not sure if this would be meaningfully beneficial in practice.

Reply:
@kerberizer We'd never end up with partial failovers as a single failing will fail the entire build, so the order would only change how early we abort a build, not whether we abort a build at all